### PR TITLE
Late June pull 2

### PR DESCRIPTION
**Pull request by DandelionSprout**
* Added 27 lists, after it turned out that the Little Snitch community has become considerably more active recently.
* Added more tags to the *Unified Hosts* lists.

### CODE REVIEW

**Comment by DandelionSprout** [comment]
The build error appears to me to just be server-side problems, thankfully.

File: data/FilterList.json [hunk 1]
```diff
@@ -17106,5 +17106,263 @@
...
     "viewUrl": "https://raw.githubusercontent.com/Laicure/HostsY_hosts/master/shithosts"
-  }
-]
\ No newline at end of file
+  },
+        {
+            "id": 1608,
+            "name": "Unified Hosts (Little Snitch)",
+            "description": "This repo provide rule groups for Little Snitch based on unified host list to block ads, malware, fake news and porn",
+            "homeUrl": "https://github.com/naveednajam/Little-Snitch---Rule-Groups,
```
Comment (collinbarrett) [inline data/FilterList.json]: Missing closing quotes here.

See: https://dev.azure.com/collinbarrett/FilterLists/_build/results?buildId=598&view=ms.vss-test-web.build-test-results-tab&runId=1000096&resultId=100000&paneView=debug

Comment (DandelionSprout) [reply data/FilterList.json]: Ah, thanks. Should be fixed now.

**Comment by collinbarrett** [comment]
Awesome. Great PR. Thanks.

**Pull request merged**